### ISSUE TITLE
resonance rod

### ISSUE THREAD

**Issue by najlitarvan** [klikli-dev/occultism#172]
divination rod that can taget anything and would not look for other world materials,
would be gained after obtaining access to afrits and will likely have 2x cooldown

**Comment by theboo** [comment]
I don't think it needs to be all the way to afrit... But maybe binding a djinni miner into the rod would make sense?

**Comment by najlitarvan** [comment]
well its upto you but around djinni or afrit would make sence
